tests/unit/test_normalizer.py

```diff
@@ -10,6 +10,25 @@
 cur_dir = os.path.dirname(os.path.abspath(__file__))
 
 
+@pytest.fixture
+def test_image_uri_doc():
+    return Document(uri=os.path.join(cur_dir, '..', 'data', 'test_image.png'))
```

Comment:
how do you think instead of create a real image, we "generate" a image from numpy and save it in `tmpdir` pytest fixture? Then you get rid of all the test path stuff

Reply:
Sure, please see [cdb652d](https://github.com/jina-ai/executor-image-normalizer/pull/11/commits/cdb652d7958ee633b9fb438efde11b7d66d2bbba) for the change.